sktime/forecasting/varmax.py

```diff
@@ -0,0 +1,312 @@
+            enforce_stationarity = self.enforce_stationarity,
+            enforce_invertibility = self.enforce_invertibility,
+            trend_offset = self.trend_offset,
+        )
+        self._fitted_forecaster = self._forecaster.fit(
+            start_params = self.start_params,
+            transformed = self.transformed,
+            includes_fixed = self.includes_fixed,
+            cov_type = self.cov_type,
+            cov_kwds = self.cov_kwds,
+            method = self.method,
+            maxiter = self.maxiter,
+            full_output = self.full_output,
+            disp = self.disp,
+            callback = self.callback,
+            return_params = self.return_params,
+            optim_score = self.optim_score,
+            optim_complex_step = self.optim_complex_step,
+            optim_hessian = self.optim_hessian,
+            flags = self.flags,
+            low_memory = self.low_memory,
+        )
+        return self
+
+    def _predict(self, fh, X=None):
+        """
+        Wrap Statmodel's VARMAX forecast method.
+
+        Parameters
+        ----------
+        fh : ForecastingHorizon
+            The forecasters horizon with the steps ahead to to predict.
+            Default is one-step ahead forecast,
+            i.e. np.array([1])
+        X : pd.DataFrame, optional (default=None)
+            Exogenous variables are ignored.
+
+        Returns
+        -------
+        y_pred : np.ndarray
+            Returns series of predicted values.
+        """
+        start, end = fh.to_absolute_int(self._y.index[0], self.cutoff)[[0, -1]]
+
+        return self._fitted_forecaster.predict(start = start,
+            end = end,
+            dynamic = self.dynamic,
+            information_set = self.information_set,
+            signal_only = self.signal_only,
+            exog = X,
+        )
+
+
+    @classmethod
+    def get_test_params(cls, parameter_set="default"):
+        """Return testing parameter settings for the estimator.
+
+        Parameters
+        ----------
+        parameter_set : str, default="default"
+            Name of the set of test parameters to return, for use in tests. If no
+            special parameters are defined for a value, will return `"default"` set.
+            There are currently no reserved values for forecasters.
+
+        Returns
+        -------
+        params : dict or list of dict, default = {}
+            Parameters to create testing instances of the class
+            Each dict are parameters to construct an "interesting" test instance, i.e.,
+            `MyClass(**params)` or `MyClass(**params[i])` creates a valid test instance.
+            `create_test_instance` uses the first (or only) dictionary in `params`
+        """
+
+        # todo: set the testing parameters for the estimators
+        # Testing parameters can be dictionary or list of dictionaries.
+        # Testing parameter choice should cover internal cases well.
```

Comment:
this needs to be done. simplest option is just to have an empty dictionary (so you this class will only be tested on the defaults). but would be good to include two or three options, depending on if there are any edge cases in VARMAX

Reply:
quick question @Lovkush-A, do the unit tests iterate through all of these parameter sets?

Reply:
@KatieBuc The test suites (e.g. test_all_forecasters and test_all_estimators) will iterate through all of these parameter sets.

The unit tests you have written won't (unless you write code to loop through these parameters)

Reply:
See this method, which creates list of instances: https://github.com/alan-turing-institute/sktime/blob/7be01f62e580db77da1420823291d5d03675b45d/sktime/base/_base.py#L351

Reply:
Hmm okay. I'll put this as a question for today's catch up. 
`params = [
            {"order": (0, 0)},
            {"order": (1, 0)},
            {"order": (0, 1)},
            {"order": (1, 1)},
        ]
'
for `order = (p,q)` but when `q=1`, then `X` must be populated (by default `X=None`)

Reply:
Wait, ignore me that was wrong^^